GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -193,6 +180,37 @@ string GetSubmoduleItemImage(DetailedSubmoduleInfo details)
                     return details.IsDirty ? nameof(Images.SubmoduleDirty) : nameof(Images.FileStatusModified);
                 }
             }
+
+            internal async Task<bool> SetStatusToolTipAsync(CancellationToken token)
```

Comment:
Do you have plans for further usage? Else:
```suggestion
            private async Task SetStatusToolTipAsync(CancellationToken token)
```

Reply:
ok

Reply:
Why not `private`?
The final `return` is superfluous.

Reply:
removing the last return

`internal` as it is called from SubmoduleTree